packs/swift/chart/values.yaml

```diff
@@ -0,0 +1,22 @@
+# Default values for swift.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 2
+image:
+  registry: docker.io
+  org: nomisbeme
+  name: swift-onbuild-bitnami
+  tag: 3.0.2.1
+  pullPolicy: Always
```

Comment:
this should be changed to `ifNotPresent` as seen [in other charts](https://github.com/Azure/draft/blob/master/packs/golang/chart/values.yaml#L10).